src/Chapter18.Tests/Listing18.02.Tests.cs

```diff
@@ -0,0 +1,17 @@
+﻿using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace AddisonWesley.Michaelis.EssentialCSharp.Chapter18.Listing18_02.Tests
+{
+    [TestClass]
+    public class ProgramTests
+    {
+        [TestMethod]
+        public void MainTest()
+        {
+            const string expected = @"";
+
+            IntelliTect.TestTools.Console.ConsoleAssert.Expect(
+                expected, Program.Main);
```

Comment:
What is the goal of this test? Is it supposed to return nothing or is there something better we can test?

Reply:
I looked at the listing 18.02 and I don't think we actually need a unit test for it